src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -14,54 +13,75 @@
 import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static subway.StationSteps.*;
 
 @DisplayName("지하철역 관련 기능")
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
-public class StationAcceptanceTest {
+public class StationAcceptanceTest extends ApiTest {
     /**
      * When 지하철역을 생성하면
      * Then 지하철역이 생성된다
      * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
      */
-    @DisplayName("지하철역을 생성한다.")
+    @DisplayName("지하철역을 생성한다")
     @Test
     void createStation() {
         // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        String stationName = "강남역";
+        StationRequest request = 지하철역_생성_요청(stationName);
+        ExtractableResponse<Response> createStationResponse = 지하철역_생성(request);
```

Comment:
```suggestion
        var 강남역 = 지하철역_생성("강남역");
```

저는 인수테스트는 직관적이어야 한다고 생각합니다. 그 이유는 인수테스트를 보는 사람은 프론트엔드 개발자, API를 활용하는 사람이기 때문입니다. 이러한 사용자들은 개발자일 수도 있고, 비 개발자일 수도 있습니다. 이러한 사용자에게 보여주기 위해서는 직관적이게 이 테스트는 이러한 시나리오를 가지고 있다는 것을 좀 더 알기 쉽게 표현해야 한다고 생각합니다. 그래서, 저는 인수 테스트에 대해서 타입 추론을 하는 것을 추천드리며, request도 최대한 안으로 숨기도록 하고 있습니다.

Reply:
안녕하세요. 2단계 반영 전에 질문하고 싶은 내용이 있어서 코멘트남겨두겠습니다~!😁

> 저는 인수테스트는 직관적이어야 한다고 생각합니다.

말씀주신 내용 공감합니다!
- 이에 따라 2단계에서는 변수명 변경 및 타입 추론을 적용해보겠습니다


추가로 제안주신 코드에서 이야기해보고 싶은 부분이 있어 내용남깁니다!😊
- 제안주신 스니펫에서는 request 부분을 생략하고 String 단일 파라미터로 변경해주셨는데요.
- 저는 지하철역 생성이라는 피쳐를 고려했을 때 이름 외에도 추가로 파라미터로 받을 수 있는 상황이 발생할 수 있을 것 같아서 API 요청 상 사용하는 request body 클래스를 인자로 받는 형태로 작성했습니다. (어찌보면 사용자 또는 관리자가 입력하는 폼이라고 생각할 수도 있겠네요)
- 어찌보면 직관적이지 않고, 과한 확장성을 고려한 것일 수도 있는데 이 부분(직관성 vs 확징성)에 대해서 플람께서는 어떤 가치 판단 또는 기준이 있으신 궁금합니다~!

Reply:
확정성을 위하여 requestBody를 사용하셨군요. 확장성을 위해서라면, requestBody를 사용하는 것에 충분히 이해합니다. 나중에 stationName 뿐만 아니라 다른 것도 들어갈 수도 있죠. 

이에 대해서 확장성이 미래에 보인다면, request를 새로 만들어주는 방향으로 해주시면 되며, 저 같은 경우에는 요번에 코드리뷰 했듯이 직관적으로 표현한 이유는 확장성에 대한 부분은 추후에 고려해 봐도 괜찮지 않을까 했습니다. 즉 최대한 단순하고 직관적이게 만들고 요청이 변경되었을 때는 리팩토링을 통하여 자연스럽게 만들어주는 방향을 선택했습니다. 😊

코드의 직관성 vs 확장성 둘 중에 하나를 선택하여 희동님 스타일에 따라서 미션을 진행해 보시죠 👍  이에 대해서 저는 저 나름대로 피드백을 드려볼게요! 🙇🏻